### DISCUSSION TITLE
[Project Beta] Keyboard shortcuts not working

### DISCUSSION THREAD

**Discussion by amitgupta6** [community/community#6610] [Projects and Issues]
First, really glad to see this new version of Project here in Github itself. I have used previous Project feature extensively and I loved the way you can move your cards up and down with keyboard shortcuts. Either I am missing on how to use it. I tried Shift+/ and I can see a popup which shows keyboard shortcuts. I can select a card using Shift+Space, however, I can not move it. I tried "j" and "k" but that starts editing the contents of the column. Up and down also doesn't work. It would be great to have full keyboard shortcuts like we had in earlier Projects feature.

**Comment by pyrello** [comment]
Not an answer: The lack of this feature is preventing us from adopting the new project boards. Very excited for this to eventually get added!

**Comment by arvindhsundar** [comment]
Is there any update on this? (I'm new to github)
